### ISSUE TITLE
dynamic gesture

### ISSUE THREAD

**Issue by hanako94** [EvilPort2/Sign-Language#5]
Hi, may i know how to modify this so it can recognize dynamic sign gesture as well?

**Comment by s9k96** [comment]
I think the application is recognizing gestures dynamically (gestures can be changed in the video), though that gesture should be stored in the database and also trained on the neural net.

**Comment by adharshgec** [comment]
I think what he meant was, to recognize dynamic gestures such as J and Z... You need to go for 3d conv nets and train using video datasets for that.

**Comment by hanako94** [comment]
I think 3d conv nets can recognize only static gesture or image right? for dynamic need to use lstm or rnn which can store temporal features

**Comment by adharshgec** [comment]
2d convnets are used for static getsures, 3d convets can be used to learn spatio-temporal features. RNN's can also be used, but I don't know much about that.

**Comment by hanako94** [comment]
i see, may i know after i trained a keras model(hdf5), is that possible to deploy it to android phone for recognition? i need to convert keras to tf model?

**Comment by adharshgec** [comment]
Yeah you may need to convert to tf model and use the Tensorflow Java API

**Comment by hanako94** [comment]
thanks~ i got a problem here, since i need to recognize dynamic gesture, then i might want to store n frames for prediction,  i will use  keras library (keras.preprocessing.image import img_to_array, load_img) when predicting the gesture, but tf api doesn't have this function. so what should i do for this part?

**Comment by EvilPort2** [comment]
whether you use RNNs or LSTMs, how is the dataset going to look like?

**Comment by hanako94** [comment]
im using lstm, dataset is image frames from video clips
